### ISSUE TITLE
Support accent mark and umlaut character ('¨) while writing

### ISSUE THREAD

**Issue by ppnon** [maoschanz/notes-extension-gnome#61]
When the accent or umlaut key is pressed, the corresponding character is not written in the note. But they can be copied into the note without problems.

![Peek 2021-02-12 14-55](https://user-images.githubusercontent.com/7077297/107819380-0e013d80-6d47-11eb-98b2-187ac9226246.gif)

System:
GNOME Shell 3.36.4
Pop!_OS 20.04 LTS x86_64
LANG=en_US.UTF-8

**Comment by maoschanz** [comment]
sorry that's sadly not my problem, but a GNOME Shell bug. You can try to type in the overview search entry it'll do the same. Or in the alt+f2 dialog. It's event more ridiculous with ÉÈÀÇ etc. where the character is typed... but in lowercase

**Comment by afterlastangel** [comment]
I can confirm this problem because of ibus and gnome-shell. I can use the note with English only. But there is another problem, that input source option doesn't be applied for the note. So even if the last time I use English with Note, then I switch to Chrome, then switch back to note. The ibus keeps the last Input Method.  I think if you can make the note as an application it would work. ![image](https://user-images.githubusercontent.com/1694449/107921759-895e2b80-6fa1-11eb-9842-c104f94be943.png)

An application supports input method is https://github.com/konkor/cpufreq

![image](https://user-images.githubusercontent.com/1694449/107922237-51a3b380-6fa2-11eb-828d-8fb5997c8ceb.png)

**Comment by maoschanz** [comment]
Yes this cpufreq application is an application so it behaves like any normal application... 🤔 what's your point, here we're talking about an extension, i'll not throw my entire code away and start a new project from zero just because GNOME Shell has a bug regarding accentuated characters.

The only way to make GNOME Shell work is to fix GNOME Shell, not to shun it
